admin-ui/app/dialogs/create-app.html

```diff
@@ -3,28 +3,30 @@
         <button type="button" class="close" aria-hidden="true" ng-click="cancel()">
             <span class="pficon pficon-close"></span>
         </button>
-        <h4 class="modal-title">{{ application.id ? 'Edit' : 'Create' }} Application</h4>
+        <h4 class="modal-title" ng-if="application.id" translate>Edit Application</h4>
+        <h4 class="modal-title" ng-if="!application.id" translate>Create Application</h4>
     </div>
     <div class="modal-body">
         <div class="form-horizontal">
             <div class="form-group">
-                <label class="col-sm-4 control-label" for="createAppName">Name</label>
+                <label class="col-sm-4 control-label" for="createAppName" translate>Name</label>
 
                 <div class="col-sm-6">
-                    <input type="text" placeholder="e.g. MyApp" id="createAppName" class="form-control" ng-model="application.name" required pf-focused="true">
+                    <input type="text" data-placeholder="'e.g. MyApp'|translate" id="createAppName" class="form-control" ng-model="application.name" required pf-focused="true">
                 </div>
             </div>
             <div class="form-group">
-                <label class="col-sm-4 control-label" for="createAppDescription">Description</label>
+                <label class="col-sm-4 control-label" for="createAppDescription" translate>Description</label>
 
                 <div class="col-sm-6">
-                    <input type="text" placeholder="e.g. A useful application for everyone." id="createAppDescription" class="form-control" ng-model="application.description">
+                    <input type="text" data-placeholder="'e.g. A useful application for everyone.'|translate" id="createAppDescription" class="form-control" ng-model="application.description">
                 </div>
             </div>
         </div>
     </div>
     <div class="modal-footer">
-        <input type="button" class="btn btn-default" ng-click="cancel()" value="Cancel" />
-        <input type="submit" class="btn btn-primary" ng-click="ok(application)" ng-disabled="form.$invalid" value="{{ application.id ? 'Save' : 'Create' }}" />
+        <input type="button" class="btn btn-default" ng-click="cancel()" data-value="'Cancel'|translate" />
+        <input type="submit" class="btn btn-primary" ng-click="ok(application)" ng-disabled="form.$invalid" ng-if="application.id" data-value="'Save'|translate" />
+        <input type="submit" class="btn btn-primary" ng-click="ok(application)" ng-disabled="form.$invalid" ng-if="!application.id" data-value="'Create'|translate" />
```

Comment:
What about:
```html
<input ... data-value="{{ application.id ? 'Save' : 'Create' | translate }}" />
```
instead of 
```html
<input ... ng-if="application.id" data-value="'Save'|translate" />
<input ... ng-if="!application.id" data-value="'Create'|translate" />
```
It looks cleaner IMO
Ref: https://github.com/angular-translate/grunt-angular-translate/pull/69